tools.yaml

```diff
@@ -4,17 +4,21 @@ install_resolver_dependencies: true
 install_tool_dependencies: false
 
 tools:
+  - name: suite_mimodd
```

Comment:
mimodd_main

would be better than suite_mimodd (the suite really brings in only stuff that's unnecessary on a well-equipped server; you don't need another aligner, nor another SnpEff wrapper I would say)

Reply:
Mh, that is confusing ;)

Reply:
Changed it on master.

Reply:
perfect, thanks.